nwbwidgets/misc.py

```diff
@@ -285,10 +289,10 @@ def __init__(
             self.trials = trials
 
         if unit_controller is None:
-            unit_ids = units.id.data[:]
-            n_units = len(unit_ids)
+            self.unit_ids = units.id.data[:]
```

Comment:
The unit ids are  used across more than one function in this class, so I just set it as an attribute for a single IO operation